### ISSUE TITLE
Please release new version with `libc` update to `2.19-0ubuntu6.7`

### ISSUE THREAD

**Issue by newhoggy** [phusion/baseimage-docker#289]
Please see the following security advisory:

http://www.ubuntu.com/usn/usn-2900-1/

**Comment by newhoggy** [comment]
The following snippet shows that I need to run a separate `update` and `upgrade` in order to get patched `glibc`:

```
11:45 $ docker run -ti phusion/baseimage:0.9.18 bash
root@359f1c4c03a2:/# /lib/x86_64-linux-gnu/libc.so.6
GNU C Library (Ubuntu EGLIBC 2.19-0ubuntu6.6) stable release version 2.19, by Roland McGrath et al.
Copyright (C) 2014 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.
Compiled by GNU CC version 4.8.2.
Compiled on a Linux 3.13.11 system on 2015-02-25.
Available extensions:
	crypt add-on version 2.1 by Michael Glad and others
	GNU Libidn by Simon Josefsson
	Native POSIX Threads Library by Ulrich Drepper et al
	BIND-8.2.3-T5B
libc ABIs: UNIQUE IFUNC
For bug reporting instructions, please see:
<https://bugs.launchpad.net/ubuntu/+source/eglibc/+bugs>.
root@359f1c4c03a2:/# apt-get update
Ign http://archive.ubuntu.com trusty InRelease
Get:1 http://archive.ubuntu.com trusty-updates InRelease [65.9 kB]
Get:2 http://archive.ubuntu.com trusty-security InRelease [65.9 kB]
Get:3 http://archive.ubuntu.com trusty Release.gpg [933 B]
Get:4 http://archive.ubuntu.com trusty-updates/main Sources [326 kB]
Get:5 http://archive.ubuntu.com trusty-updates/restricted Sources [5217 B]
Get:6 http://archive.ubuntu.com trusty-updates/universe Sources [189 kB]
Get:7 http://archive.ubuntu.com trusty-updates/main amd64 Packages [897 kB]
Get:8 http://archive.ubuntu.com trusty-updates/restricted amd64 Packages [23.5 kB]
Get:9 http://archive.ubuntu.com trusty-updates/universe amd64 Packages [439 kB]
Get:10 http://archive.ubuntu.com trusty-security/main Sources [132 kB]
Get:11 http://archive.ubuntu.com trusty-security/restricted Sources [3920 B]
Get:12 http://archive.ubuntu.com trusty-security/universe Sources [38.2 kB]
Get:13 http://archive.ubuntu.com trusty-security/multiverse Sources [2548 B]
Get:14 http://archive.ubuntu.com trusty-security/main amd64 Packages [530 kB]
Get:15 http://archive.ubuntu.com trusty-security/restricted amd64 Packages [20.2 kB]
Get:16 http://archive.ubuntu.com trusty-security/universe amd64 Packages [161 kB]
Get:17 http://archive.ubuntu.com trusty-security/multiverse amd64 Packages [4852 B]
Get:18 http://archive.ubuntu.com trusty Release [58.5 kB]
Get:19 http://archive.ubuntu.com trusty/main Sources [1335 kB]
Get:20 http://archive.ubuntu.com trusty/restricted Sources [5335 B]
Get:21 http://archive.ubuntu.com trusty/universe Sources [7926 kB]
Get:22 http://archive.ubuntu.com trusty/main amd64 Packages [1743 kB]
Get:23 http://archive.ubuntu.com trusty/restricted amd64 Packages [16.0 kB]
Get:24 http://archive.ubuntu.com trusty/universe amd64 Packages [7589 kB]
Fetched 21.6 MB in 57s (374 kB/s)
Reading package lists... Done
root@359f1c4c03a2:/# apt-get upgrade libc6:amd64
Reading package lists... Done
Building dependency tree
Reading state information... Done
Calculating upgrade... Done
The following packages will be upgraded:
  apt apt-transport-https apt-utils base-files ca-certificates coreutils cpio
  curl gcc-4.8-base gcc-4.9-base ifupdown initscripts isc-dhcp-client
  isc-dhcp-common libapt-inst1.5 libapt-pkg4.12 libc-bin libc6 libcurl3
  libcurl3-gnutls libdrm2 libgcc1 libgcrypt11 libgnutls-openssl27 libgnutls26
  libpng12-0 libssl1.0.0 libstdc++6 libsystemd-daemon0 libudev1 login
  multiarch-support ntpdate openssh-client openssh-server openssh-sftp-server
  openssl passwd python-apt-common python3-apt python3-software-properties
  software-properties-common sysv-rc sysvinit-utils udev unattended-upgrades
46 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 16.6 MB of archives.
After this operation, 52.2 kB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://archive.ubuntu.com/ubuntu/ trusty-updates/main base-files amd64 7.2ubuntu5.4 [67.6 kB]
Get:2 http://archive.ubuntu.com/ubuntu/ trusty-updates/main coreutils amd64 8.21-1ubuntu5.3 [1090 kB]
Get:3 http://archive.ubuntu.com/ubuntu/ trusty-updates/main login amd64 1:4.1.5.1-1ubuntu9.2 [301 kB]
Get:4 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libc-bin amd64 2.19-0ubuntu6.7 [1169 kB]
Get:5 http://archive.ubuntu.com/ubuntu/ trusty-updates/main gcc-4.9-base amd64 4.9.3-0ubuntu4 [15.1 kB]
Get:6 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libgcc1 amd64 1:4.9.3-0ubuntu4 [39.2 kB]
Get:7 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libc6 amd64 2.19-0ubuntu6.7 [4719 kB]
Get:8 http://archive.ubuntu.com/ubuntu/ trusty-updates/main gcc-4.8-base amd64 4.8.4-2ubuntu1~14.04.1 [16.0 kB]
Get:9 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libstdc++6 amd64 4.8.4-2ubuntu1~14.04.1 [259 kB]
Get:10 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libapt-pkg4.12 amd64 1.0.1ubuntu2.11 [637 kB]
Get:11 http://archive.ubuntu.com/ubuntu/ trusty-updates/main apt amd64 1.0.1ubuntu2.11 [952 kB]
Get:12 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libapt-inst1.5 amd64 1.0.1ubuntu2.11 [58.6 kB]
Get:13 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libgcrypt11 amd64 1.5.3-2ubuntu4.3 [239 kB]
Get:14 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libgnutls-openssl27 amd64 2.12.23-12ubuntu2.5 [18.4 kB]
Get:15 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libgnutls26 amd64 2.12.23-12ubuntu2.5 [393 kB]
Get:16 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libssl1.0.0 amd64 1.0.1f-1ubuntu2.18 [830 kB]
Get:17 http://archive.ubuntu.com/ubuntu/ trusty-updates/main ntpdate amd64 1:4.2.6.p5+dfsg-3ubuntu2.14.04.8 [57.2 kB]
Get:18 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libdrm2 amd64 2.4.64-1~ubuntu14.04.1 [25.4 kB]
Get:19 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libpng12-0 amd64 1.2.50-1ubuntu2.14.04.2 [118 kB]
Get:20 http://archive.ubuntu.com/ubuntu/ trusty-updates/main sysvinit-utils amd64 2.88dsf-41ubuntu6.3 [52.9 kB]
Get:21 http://archive.ubuntu.com/ubuntu/ trusty-updates/main sysv-rc all 2.88dsf-41ubuntu6.3 [36.6 kB]
Get:22 http://archive.ubuntu.com/ubuntu/ trusty-updates/main passwd amd64 1:4.1.5.1-1ubuntu9.2 [759 kB]
Get:23 http://archive.ubuntu.com/ubuntu/ trusty-updates/main initscripts amd64 2.88dsf-41ubuntu6.3 [27.7 kB]
Get:24 http://archive.ubuntu.com/ubuntu/ trusty-updates/main udev amd64 204-5ubuntu20.18 [734 kB]
Get:25 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libudev1 amd64 204-5ubuntu20.18 [33.5 kB]
Get:26 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libcurl3-gnutls amd64 7.35.0-1ubuntu2.6 [165 kB]
Get:27 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libsystemd-daemon0 amd64 204-5ubuntu20.18 [10.0 kB]
Get:28 http://archive.ubuntu.com/ubuntu/ trusty-updates/main curl amd64 7.35.0-1ubuntu2.6 [123 kB]
Get:29 http://archive.ubuntu.com/ubuntu/ trusty-updates/main libcurl3 amd64 7.35.0-1ubuntu2.6 [172 kB]
Get:30 http://archive.ubuntu.com/ubuntu/ trusty-updates/main multiarch-support amd64 2.19-0ubuntu6.7 [4488 B]
Get:31 http://archive.ubuntu.com/ubuntu/ trusty-updates/main apt-utils amd64 1.0.1ubuntu2.11 [172 kB]
Get:32 http://archive.ubuntu.com/ubuntu/ trusty-updates/main ifupdown amd64 0.7.47.2ubuntu4.3 [53.0 kB]
Get:33 http://archive.ubuntu.com/ubuntu/ trusty-updates/main isc-dhcp-client amd64 4.2.4-7ubuntu12.4 [640 kB]
Get:34 http://archive.ubuntu.com/ubuntu/ trusty-updates/main isc-dhcp-common amd64 4.2.4-7ubuntu12.4 [708 kB]
Get:35 http://archive.ubuntu.com/ubuntu/ trusty-updates/main cpio amd64 2.11+dfsg-1ubuntu1.2 [73.8 kB]
Get:36 http://archive.ubuntu.com/ubuntu/ trusty-updates/main apt-transport-https amd64 1.0.1ubuntu2.11 [25.0 kB]
Get:37 http://archive.ubuntu.com/ubuntu/ trusty-updates/main openssl amd64 1.0.1f-1ubuntu2.18 [489 kB]
Get:38 http://archive.ubuntu.com/ubuntu/ trusty-updates/main ca-certificates all 20160104ubuntu0.14.04.1 [190 kB]
Get:39 http://archive.ubuntu.com/ubuntu/ trusty-updates/main openssh-sftp-server amd64 1:6.6p1-2ubuntu2.6 [34.2 kB]
Get:40 http://archive.ubuntu.com/ubuntu/ trusty-updates/main openssh-server amd64 1:6.6p1-2ubuntu2.6 [321 kB]
Get:41 http://archive.ubuntu.com/ubuntu/ trusty-updates/main openssh-client amd64 1:6.6p1-2ubuntu2.6 [562 kB]
Get:42 http://archive.ubuntu.com/ubuntu/ trusty-updates/main python-apt-common all 0.9.3.5ubuntu2 [16.6 kB]
Get:43 http://archive.ubuntu.com/ubuntu/ trusty-updates/main python3-apt amd64 0.9.3.5ubuntu2 [139 kB]
Get:44 http://archive.ubuntu.com/ubuntu/ trusty-updates/main software-properties-common all 0.92.37.7 [9356 B]
Get:45 http://archive.ubuntu.com/ubuntu/ trusty-updates/main unattended-upgrades all 0.82.1ubuntu2.4 [25.9 kB]
Get:46 http://archive.ubuntu.com/ubuntu/ trusty-updates/main python3-software-properties all 0.92.37.7 [19.1 kB]
Fetched 16.6 MB in 51s (322 kB/s)
Extracting templates from packages: 100%
Preconfiguring packages ...
(Reading database ... 13649 files and directories currently installed.)
Preparing to unpack .../base-files_7.2ubuntu5.4_amd64.deb ...
Unpacking base-files (7.2ubuntu5.4) over (7.2ubuntu5.3) ...
Setting up base-files (7.2ubuntu5.4) ...
Installing new version of config file /etc/issue ...
Installing new version of config file /etc/issue.net ...
Installing new version of config file /etc/lsb-release ...
Installing new version of config file /etc/os-release ...
(Reading database ... 13649 files and directories currently installed.)
Preparing to unpack .../coreutils_8.21-1ubuntu5.3_amd64.deb ...
Unpacking coreutils (8.21-1ubuntu5.3) over (8.21-1ubuntu5.1) ...
Setting up coreutils (8.21-1ubuntu5.3) ...
(Reading database ... 13649 files and directories currently installed.)
Preparing to unpack .../login_1%3a4.1.5.1-1ubuntu9.2_amd64.deb ...
Unpacking login (1:4.1.5.1-1ubuntu9.2) over (1:4.1.5.1-1ubuntu9.1) ...
Setting up login (1:4.1.5.1-1ubuntu9.2) ...
(Reading database ... 13649 files and directories currently installed.)
Preparing to unpack .../libc-bin_2.19-0ubuntu6.7_amd64.deb ...
Unpacking libc-bin (2.19-0ubuntu6.7) over (2.19-0ubuntu6.6) ...
Preparing to unpack .../libc6_2.19-0ubuntu6.7_amd64.deb ...
Unpacking libc6:amd64 (2.19-0ubuntu6.7) over (2.19-0ubuntu6.6) ...
Preparing to unpack .../libgcc1_1%3a4.9.3-0ubuntu4_amd64.deb ...
Unpacking libgcc1:amd64 (1:4.9.3-0ubuntu4) over (1:4.9.1-0ubuntu1) ...
Preparing to unpack .../gcc-4.9-base_4.9.3-0ubuntu4_amd64.deb ...
Unpacking gcc-4.9-base:amd64 (4.9.3-0ubuntu4) over (4.9.1-0ubuntu1) ...
Setting up gcc-4.9-base:amd64 (4.9.3-0ubuntu4) ...
Setting up libgcc1:amd64 (1:4.9.3-0ubuntu4) ...
Setting up libc6:amd64 (2.19-0ubuntu6.7) ...
Setting up libc-bin (2.19-0ubuntu6.7) ...
(Reading database ... 13649 files and directories currently installed.)
Preparing to unpack .../gcc-4.8-base_4.8.4-2ubuntu1~14.04.1_amd64.deb ...
Unpacking gcc-4.8-base:amd64 (4.8.4-2ubuntu1~14.04.1) over (4.8.4-2ubuntu1~14.04) ...
Setting up gcc-4.8-base:amd64 (4.8.4-2ubuntu1~14.04.1) ...
(Reading database ... 13649 files and directories currently installed.)
Preparing to unpack .../libstdc++6_4.8.4-2ubuntu1~14.04.1_amd64.deb ...
Unpacking libstdc++6:amd64 (4.8.4-2ubuntu1~14.04.1) over (4.8.4-2ubuntu1~14.04) ...
Setting up libstdc++6:amd64 (4.8.4-2ubuntu1~14.04.1) ...
Processing triggers for libc-bin (2.19-0ubuntu6.7) ...
(Reading database ... 13649 files and directories currently installed.)
Preparing to unpack .../libapt-pkg4.12_1.0.1ubuntu2.11_amd64.deb ...
Unpacking libapt-pkg4.12:amd64 (1.0.1ubuntu2.11) over (1.0.1ubuntu2.10) ...
Setting up libapt-pkg4.12:amd64 (1.0.1ubuntu2.11) ...
Processing triggers for libc-bin (2.19-0ubuntu6.7) ...
(Reading database ... 13649 files and directories currently installed.)
Preparing to unpack .../apt_1.0.1ubuntu2.11_amd64.deb ...
Unpacking apt (1.0.1ubuntu2.11) over (1.0.1ubuntu2.10) ...
Setting up apt (1.0.1ubuntu2.11) ...
Processing triggers for libc-bin (2.19-0ubuntu6.7) ...
(Reading database ... 13649 files and directories currently installed.)
Preparing to unpack .../libapt-inst1.5_1.0.1ubuntu2.11_amd64.deb ...
Unpacking libapt-inst1.5:amd64 (1.0.1ubuntu2.11) over (1.0.1ubuntu2.10) ...
Preparing to unpack .../libgcrypt11_1.5.3-2ubuntu4.3_amd64.deb ...
Unpacking libgcrypt11:amd64 (1.5.3-2ubuntu4.3) over (1.5.3-2ubuntu4.2) ...
Preparing to unpack .../libgnutls-openssl27_2.12.23-12ubuntu2.5_amd64.deb ...
Unpacking libgnutls-openssl27:amd64 (2.12.23-12ubuntu2.5) over (2.12.23-12ubuntu2.3) ...
Preparing to unpack .../libgnutls26_2.12.23-12ubuntu2.5_amd64.deb ...
Unpacking libgnutls26:amd64 (2.12.23-12ubuntu2.5) over (2.12.23-12ubuntu2.3) ...
Preparing to unpack .../libssl1.0.0_1.0.1f-1ubuntu2.18_amd64.deb ...
Unpacking libssl1.0.0:amd64 (1.0.1f-1ubuntu2.18) over (1.0.1f-1ubuntu2.16) ...
Preparing to unpack .../ntpdate_1%3a4.2.6.p5+dfsg-3ubuntu2.14.04.8_amd64.deb ...
Unpacking ntpdate (1:4.2.6.p5+dfsg-3ubuntu2.14.04.8) over (1:4.2.6.p5+dfsg-3ubuntu2.14.04.5) ...
Preparing to unpack .../libdrm2_2.4.64-1~ubuntu14.04.1_amd64.deb ...
Unpacking libdrm2:amd64 (2.4.64-1~ubuntu14.04.1) over (2.4.60-2~ubuntu14.04.1) ...
Preparing to unpack .../libpng12-0_1.2.50-1ubuntu2.14.04.2_amd64.deb ...
Unpacking libpng12-0:amd64 (1.2.50-1ubuntu2.14.04.2) over (1.2.50-1ubuntu2.14.04.1) ...
Preparing to unpack .../sysvinit-utils_2.88dsf-41ubuntu6.3_amd64.deb ...
Unpacking sysvinit-utils (2.88dsf-41ubuntu6.3) over (2.88dsf-41ubuntu6.2) ...
Preparing to unpack .../sysv-rc_2.88dsf-41ubuntu6.3_all.deb ...
Unpacking sysv-rc (2.88dsf-41ubuntu6.3) over (2.88dsf-41ubuntu6.2) ...
Processing triggers for ureadahead (0.100.0-16) ...
Setting up sysvinit-utils (2.88dsf-41ubuntu6.3) ...
Setting up sysv-rc (2.88dsf-41ubuntu6.3) ...
(Reading database ... 13649 files and directories currently installed.)
Preparing to unpack .../passwd_1%3a4.1.5.1-1ubuntu9.2_amd64.deb ...
Unpacking passwd (1:4.1.5.1-1ubuntu9.2) over (1:4.1.5.1-1ubuntu9.1) ...
Processing triggers for ureadahead (0.100.0-16) ...
Setting up passwd (1:4.1.5.1-1ubuntu9.2) ...
(Reading database ... 13649 files and directories currently installed.)
Preparing to unpack .../initscripts_2.88dsf-41ubuntu6.3_amd64.deb ...
Unpacking initscripts (2.88dsf-41ubuntu6.3) over (2.88dsf-41ubuntu6.2) ...
Preparing to unpack .../udev_204-5ubuntu20.18_amd64.deb ...
Adding 'diversion of /bin/udevadm to /bin/udevadm.upgrade by fake-udev'
Unpacking udev (204-5ubuntu20.18) over (204-5ubuntu20.15) ...
Preparing to unpack .../libudev1_204-5ubuntu20.18_amd64.deb ...
Unpacking libudev1:amd64 (204-5ubuntu20.18) over (204-5ubuntu20.15) ...
Preparing to unpack .../libcurl3-gnutls_7.35.0-1ubuntu2.6_amd64.deb ...
Unpacking libcurl3-gnutls:amd64 (7.35.0-1ubuntu2.6) over (7.35.0-1ubuntu2.5) ...
Preparing to unpack .../libsystemd-daemon0_204-5ubuntu20.18_amd64.deb ...
Unpacking libsystemd-daemon0:amd64 (204-5ubuntu20.18) over (204-5ubuntu20.15) ...
Preparing to unpack .../curl_7.35.0-1ubuntu2.6_amd64.deb ...
Unpacking curl (7.35.0-1ubuntu2.6) over (7.35.0-1ubuntu2.5) ...
Preparing to unpack .../libcurl3_7.35.0-1ubuntu2.6_amd64.deb ...
Unpacking libcurl3:amd64 (7.35.0-1ubuntu2.6) over (7.35.0-1ubuntu2.5) ...
Preparing to unpack .../multiarch-support_2.19-0ubuntu6.7_amd64.deb ...
Unpacking multiarch-support (2.19-0ubuntu6.7) over (2.19-0ubuntu6.6) ...
Processing triggers for ureadahead (0.100.0-16) ...
Setting up multiarch-support (2.19-0ubuntu6.7) ...
(Reading database ... 13650 files and directories currently installed.)
Preparing to unpack .../apt-utils_1.0.1ubuntu2.11_amd64.deb ...
Unpacking apt-utils (1.0.1ubuntu2.11) over (1.0.1ubuntu2.10) ...
Preparing to unpack .../ifupdown_0.7.47.2ubuntu4.3_amd64.deb ...
Unpacking ifupdown (0.7.47.2ubuntu4.3) over (0.7.47.2ubuntu4.1) ...
Preparing to unpack .../isc-dhcp-client_4.2.4-7ubuntu12.4_amd64.deb ...
Unpacking isc-dhcp-client (4.2.4-7ubuntu12.4) over (4.2.4-7ubuntu12.3) ...
Preparing to unpack .../isc-dhcp-common_4.2.4-7ubuntu12.4_amd64.deb ...
Unpacking isc-dhcp-common (4.2.4-7ubuntu12.4) over (4.2.4-7ubuntu12.3) ...
Preparing to unpack .../cpio_2.11+dfsg-1ubuntu1.2_amd64.deb ...
Unpacking cpio (2.11+dfsg-1ubuntu1.2) over (2.11+dfsg-1ubuntu1.1) ...
Preparing to unpack .../apt-transport-https_1.0.1ubuntu2.11_amd64.deb ...
Unpacking apt-transport-https (1.0.1ubuntu2.11) over (1.0.1ubuntu2.10) ...
Preparing to unpack .../openssl_1.0.1f-1ubuntu2.18_amd64.deb ...
Unpacking openssl (1.0.1f-1ubuntu2.18) over (1.0.1f-1ubuntu2.16) ...
Preparing to unpack .../ca-certificates_20160104ubuntu0.14.04.1_all.deb ...
Unpacking ca-certificates (20160104ubuntu0.14.04.1) over (20141019ubuntu0.14.04.1) ...
Preparing to unpack .../openssh-sftp-server_1%3a6.6p1-2ubuntu2.6_amd64.deb ...
Unpacking openssh-sftp-server (1:6.6p1-2ubuntu2.6) over (1:6.6p1-2ubuntu2.3) ...
Preparing to unpack .../openssh-server_1%3a6.6p1-2ubuntu2.6_amd64.deb ...
Unpacking openssh-server (1:6.6p1-2ubuntu2.6) over (1:6.6p1-2ubuntu2.3) ...
Preparing to unpack .../openssh-client_1%3a6.6p1-2ubuntu2.6_amd64.deb ...
Unpacking openssh-client (1:6.6p1-2ubuntu2.6) over (1:6.6p1-2ubuntu2.3) ...
Preparing to unpack .../python-apt-common_0.9.3.5ubuntu2_all.deb ...
Unpacking python-apt-common (0.9.3.5ubuntu2) over (0.9.3.5ubuntu1) ...
Preparing to unpack .../python3-apt_0.9.3.5ubuntu2_amd64.deb ...
Unpacking python3-apt (0.9.3.5ubuntu2) over (0.9.3.5ubuntu1) ...
Preparing to unpack .../software-properties-common_0.92.37.7_all.deb ...
Unpacking software-properties-common (0.92.37.7) over (0.92.37.6) ...
Preparing to unpack .../unattended-upgrades_0.82.1ubuntu2.4_all.deb ...
Unpacking unattended-upgrades (0.82.1ubuntu2.4) over (0.82.1ubuntu2.3) ...
Preparing to unpack .../python3-software-properties_0.92.37.7_all.deb ...
Unpacking python3-software-properties (0.92.37.7) over (0.92.37.6) ...
Processing triggers for ureadahead (0.100.0-16) ...
Setting up libapt-inst1.5:amd64 (1.0.1ubuntu2.11) ...
Setting up libgcrypt11:amd64 (1.5.3-2ubuntu4.3) ...
Setting up libgnutls26:amd64 (2.12.23-12ubuntu2.5) ...
Setting up libgnutls-openssl27:amd64 (2.12.23-12ubuntu2.5) ...
Setting up libssl1.0.0:amd64 (1.0.1f-1ubuntu2.18) ...
Setting up ntpdate (1:4.2.6.p5+dfsg-3ubuntu2.14.04.8) ...
Installing new version of config file /etc/network/if-up.d/ntpdate ...
Setting up libdrm2:amd64 (2.4.64-1~ubuntu14.04.1) ...
Setting up libpng12-0:amd64 (1.2.50-1ubuntu2.14.04.2) ...
Setting up initscripts (2.88dsf-41ubuntu6.3) ...
Installing new version of config file /etc/init.d/ondemand ...
Setting up libudev1:amd64 (204-5ubuntu20.18) ...
Setting up udev (204-5ubuntu20.18) ...
invoke-rc.d: policy-rc.d denied execution of restart.
Removing 'diversion of /bin/udevadm to /bin/udevadm.upgrade by fake-udev'
update-initramfs: deferring update (trigger activated)
Setting up libcurl3-gnutls:amd64 (7.35.0-1ubuntu2.6) ...
Setting up libsystemd-daemon0:amd64 (204-5ubuntu20.18) ...
Setting up libcurl3:amd64 (7.35.0-1ubuntu2.6) ...
Setting up curl (7.35.0-1ubuntu2.6) ...
Setting up apt-utils (1.0.1ubuntu2.11) ...
Setting up isc-dhcp-common (4.2.4-7ubuntu12.4) ...
Setting up isc-dhcp-client (4.2.4-7ubuntu12.4) ...
Setting up cpio (2.11+dfsg-1ubuntu1.2) ...
Setting up apt-transport-https (1.0.1ubuntu2.11) ...
Setting up openssl (1.0.1f-1ubuntu2.18) ...
Setting up ca-certificates (20160104ubuntu0.14.04.1) ...
Setting up openssh-client (1:6.6p1-2ubuntu2.6) ...
Setting up openssh-sftp-server (1:6.6p1-2ubuntu2.6) ...
Setting up openssh-server (1:6.6p1-2ubuntu2.6) ...
Creating SSH2 RSA key; this may take some time ...
Creating SSH2 DSA key; this may take some time ...
Creating SSH2 ECDSA key; this may take some time ...
Creating SSH2 ED25519 key; this may take some time ...
invoke-rc.d: policy-rc.d denied execution of restart.
Setting up python-apt-common (0.9.3.5ubuntu2) ...
Setting up python3-apt (0.9.3.5ubuntu2) ...
Setting up unattended-upgrades (0.82.1ubuntu2.4) ...
Setting up python3-software-properties (0.92.37.7) ...
Setting up software-properties-common (0.92.37.7) ...
Processing triggers for ureadahead (0.100.0-16) ...
Setting up ifupdown (0.7.47.2ubuntu4.3) ...
Processing triggers for libc-bin (2.19-0ubuntu6.7) ...
Processing triggers for initramfs-tools (0.103ubuntu4.2) ...
Processing triggers for ca-certificates (20160104ubuntu0.14.04.1) ...
Updating certificates in /etc/ssl/certs... 19 added, 19 removed; done.
Running hooks in /etc/ca-certificates/update.d....done.
root@359f1c4c03a2:/# /lib/x86_64-linux-gnu/libc.so.6
GNU C Library (Ubuntu EGLIBC 2.19-0ubuntu6.7) stable release version 2.19, by Roland McGrath et al.
Copyright (C) 2014 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.
Compiled by GNU CC version 4.8.2.
Compiled on a Linux 3.13.11 system on 2016-02-16.
Available extensions:
	crypt add-on version 2.1 by Michael Glad and others
	GNU Libidn by Simon Josefsson
	Native POSIX Threads Library by Ulrich Drepper et al
	BIND-8.2.3-T5B
libc ABIs: UNIQUE IFUNC
For bug reporting instructions, please see:
<https://bugs.launchpad.net/ubuntu/+source/eglibc/+bugs>.
root@359f1c4c03a2:/#
```

**Comment by FooBarWidget** [comment]
You can upgrade without having to wait for us to release anything: https://github.com/phusion/baseimage-docker#upgrading_os

**Comment by FooBarWidget** [comment]
Merged into #286.
